### PR TITLE
Change hours worked on friday or on summertime

### PR DESCRIPTION
**Pull request by sergiolopezaplazame**

### CODE REVIEW

File: tests/test_clockin.py [hunk 1]
```diff
@@ -87,3 +88,33 @@ def test_get_national_holidays(self, login, auth_user_id, absences):
...
         assert national_holidays[0] == date(2019, 8, 19)
+
+
+class TestClockIn(object):
+
+    def test_create_friday_register(self):
+        clock_in = ClockIn(2021, 2)
+
+        with mock.patch('clockin.ClockIn._reduced_day') as mock_create_register, \
+                mock.patch.object(Absence, 'is_friday', new_callable=mock.PropertyMock, return_value=True):
```
Comment (antonioIrizar) [inline tests/test_clockin.py]: Why do you are mock is_friday? you can use a day that you know is friday

File: tests/test_clockin.py [hunk 2]
```diff
@@ -87,3 +88,33 @@ def test_get_national_holidays(self, login, auth_user_id, absences):
...
+            clock_in.one_day(1)
+
+        mock_create_register.assert_called()
+
+    def test_create_weekday_no_friday_register(self):
+        clock_in = ClockIn(2021, 2)
+
+        with mock.patch('clockin.ClockIn._weekday_no_friday') as mock_create_register, \
+                mock.patch.object(Absence, 'is_reduced_workday', new_callable=mock.PropertyMock, return_value=False):
```
Comment (antonioIrizar) [inline tests/test_clockin.py]: same like other comment about mock

File: tests/test_clockin.py [hunk 3]
```diff
@@ -87,3 +88,33 @@ def test_get_national_holidays(self, login, auth_user_id, absences):
...
+            clock_in.one_day(1)
+
+        mock_create_register.assert_called()
+
+    def test_create_register_in_summertime(self):
+        clock_in = ClockIn(2021, 2)
+
+        with mock.patch('clockin.ClockIn._reduced_day') as mock_create_register, \
+                mock.patch.object(Absence, 'is_summertime', new_callable=mock.PropertyMock, return_value=True):
```
Comment (antonioIrizar) [inline tests/test_clockin.py]: same like other comment about mock

**Review by antonioIrizar — CHANGES_REQUESTED** [review]

File: tests/test_clockin.py (hunk 1 above)
Comment (sergiolopezaplazame) [reply tests/test_clockin.py]: Changed

File: tests/test_clockin.py (hunk 2 above)
Comment (sergiolopezaplazame) [reply tests/test_clockin.py]: Changed

File: tests/test_clockin.py (hunk 3 above)
Comment (sergiolopezaplazame) [reply tests/test_clockin.py]: Changed

**Pull request merged**